contracts/modules/staking/StakingPool.sol

```diff
@@ -356,6 +356,8 @@ contract StakingPool is IStakingPool, ERC721 {
       );
     }
 
+    require(block.timestamp > nxm.isLockedForMV(msg.sender), "Staking: Senders NXM is locked for member vote");
```

Comment:
Would skip the word "Senders" to have it similar to assessment message and I'd suggest replacing "member vote" with "voting in governance"